test/test.hs

```diff
@@ -277,6 +276,21 @@ macroTests = testGroup "macroTests" $
                                          , ("c", "'xyvb'V") ] coinput)
                                  (stringToKeyMacro "'c'V"))))
          @?= "xyyzxyyyxyyzxyyyxyyzxyyyxyyzxyyy"
+     , testCase "RepeatLast test 38" $
+         snd (last (accumulateActions (unwindMacros coinput
+                                        (stringToKeyMacro "'xv'V"))))
+         @?= "xxxx"
+     , testCase "RepeatLast test 39" $
+         fst <$> unwindMacros coinput (stringToKeyMacro "'xv'V")
+         @?= [[(Right   "", "'xv'V",  "")],
+              [(Left    "",  "xv'V",  "")],
+              [(Left   "x",   "v'V", "x")],
+              [(Left   "x",   "x'V", "x")],
```

Comment:
Before we would get here `[(Left "vx", "x'V", "x")]` .

Reply:
Wow, that's so tricky. Well spotted.